reflex/app.py

```diff
@@ -583,10 +583,30 @@ def _app_root(self, app_wrappers):
             parent = child
         return root
 
+    def _should_compile(self) -> bool:
+        """Check if the app should be compiled.
+
+        Returns:
+            Whether the app should be compiled.
+        """
+        # Check the environment variable.
+        if os.environ.get(constants.SKIP_COMPILE_ENV_VAR) == "yes":
+            return False
```

Comment:
can we just get rid of this? i don't think it was being used anywhere else

Reply:
I think this is used in other places in `reflex.py`. For example we call `_skip_compile()` in `db_init` and `migrate`. Agreed we should have only one way to do this though as this could be a bit messy to have two methods.